app/actions/org/total_stat.rb

```diff
@@ -0,0 +1,55 @@
+require './app/services/org/stat_joined_user'
```

Comment:
Avoid using `require` in Rails app files. These should be autoloaded on each new request.

If the files aren't being autoloaded, you might need to add `config.autoload_paths += [ Rails.root.join('app', 'services')]` to **application.rb**

More info in the Rails docs here:
https://guides.rubyonrails.org/autoloading_and_reloading_constants.html